console/module/job/src/main/java/org/eclipse/kapua/app/console/module/job/client/schedule/JobTabSchedules.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017 Eurotech and/or its affiliates and others
+ * Copyright (c) 2018 Eurotech and/or its affiliates and others
```

Comment:
Usualy either date is changed to last change or 2017, 2018 if file was created in 2017 and changed in 2018.